sds_data_manager/lambda_code/SDSCode/queries.py

```diff
@@ -32,7 +32,7 @@ def lambda_handler(event, context):
     # create the opensearch query from the API parameters
     query = Query(event["queryStringParameters"])
     client = _create_open_search_client()
-    logger.info("Query: " + query.query_dsl())
+    logger.info("Query: " + str(query.query_dsl()))
```

Comment:
```suggestion
    logger.info("Query: %s", str(query.query_dsl()))
```
Rather than formatting our own statements, it might be beneficial to let the printf format it for us, especially if we start logging longer statements.

(Just noticed this here, so it can be a follow-up PR to address these)

Reply:
Agreed. I'll create a new issue for this